### ISSUE TITLE
v1.9.0 Release. Max pwm on main outputs in stabilize mode with throttle stick zero.

### ISSUE THREAD

**Issue by group174311** [PX4/Firmware#12152]
In stabilized mode, on actuators outputs the maximum control signal was applied, when throttle was in zero.

**To Reproduce**
1. arm drone.

**Log Files and Screenshots**
https://logs.px4.io/plot_app?log=cf2378c0-ad28-4927-a5ec-c1afa80af0bf

**Comment by bresch** [comment]
Hi @group174311 , I found out that you set the parameters `PWM_MAIN_REV[1-2]`. I guess you wanted to reverse the rotation of the rotors. The effect of this parameter is to reverse a servo an not a motor as it inverts the PPM output: 1000us becomes 2000us, this is why the motors 1 and 2 went full throttle.
In order to reverse the direction of a motor, simply swap two phases.
I improved the description of the parameter in this PR (https://github.com/PX4/Firmware/pull/12165) to avoid confusion.
